packages/sarif-to-comment/src/cli.ts

```diff
@@ -147,6 +164,19 @@ export function run() {
         if (emptyURLReasons.length > 0) {
             console.log("Some comments were not posted, reasons will be included");
         }
+        const shouldFailResults = commentsResults.reduce((acc: boolean, result: any) => {
+            return result.shouldFail || acc; // always returns true if there is one of the map values is true
+        }, false);
```

Comment:
> always returns true if there is one of the map values is true

This behavior describe [Array.prototype.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some).

```suggestion
        const shouldFailResults = commentsResults.some(result => result.shouldFail);
```

Reply:
Done